tests/unit/Body.ts

```diff
@@ -1,92 +1,59 @@
-import { assert } from 'chai';
-import { VNode } from '@dojo/interfaces/vdom';
-import WidgetBase from '@dojo/widget-core/WidgetBase';
-import WidgetRegistry from '@dojo/widget-core/WidgetRegistry';
-import * as registerSuite from 'intern/lib/interfaces/object';
-import { SinonSpy, spy, stub } from 'sinon';
-import { cleanProperties, spyOnWidget } from '../support/util';
+import * as registerSuite from 'intern!object';
+
```

Comment:
Great, thanks for the explanation - I'm generally in favor of blank lines for grouping and clarity, just wasn't sure about the reasoning (or consistency) here.